examples/src/main/java/io/github/trimax/examples/archive/handlers/ArchiveApplicationStartupHandler.java

```diff
@@ -14,18 +14,6 @@ public void onStartup(final String[] args, final VentaContext context) {
         final var scene = context.getSceneManager().getCurrent();
         scene.setAmbientLight(new Vector3f(0.f, 0.f, 0.15f));
         context.registerArchive(Paths.get("examples", "data", "example.vea").toString());
-
-        final var cube = context.getObjectManager().create("cube", context.getObjectRepository().get("cube-without-material.json"));
-        cube.setScale(new Vector3f(5.f));
-        scene.add(cube);
-
-        final var lightXZ = context.getLightManager().create("Y light", context.getLightRepository().get("point.json"));
-        lightXZ.setPosition(new Vector3f(0.f, 5.f, 0.f));
-        lightXZ.setIntensity(1.0f);
-        scene.add(lightXZ);
-
-        final var camera = context.getCameraManager().getCurrent();
-        camera.setPosition(new Vector3f(5.f, 5.f, 5.f));
-        camera.lookAt(new Vector3f(0.f));
+        context.getTextureRegistry().get("1m.jpg");
```

Comment:
May I ask you to create an archive with two textures:
```
\folder\texture.png
\texture.png
```

and load them both? I'd like to be sure that hierarchy works as well

Reply:
Yeah, and to safe the repo space, please just create two textures in paint with size 2x2 and use them